modules/mobile/app/controllers/mobile/v0/addresses_controller.rb

```diff
@@ -15,6 +15,10 @@ def create
       end
 
       def update
+        # temporary log to track down address bug that is populating province on domestic addresses
+        Rails.logger.info('Mobile Address Update', province: address_params['province'],
+                                                   address_type: address_params['address_type'])
```

Comment:
the base address model validates that when the address_type is DOMESTIC or MILITARY, there is no province value . I was going to only have it log on those address types but I'd also like to see what international values look like.

Reply:
Unfortunately, I'm not sure this is going to be very helpful. Sentry tells us which query params are being received. 
<img width="281" alt="image" src="https://user-images.githubusercontent.com/28661253/213514473-48e777f2-cb0a-45de-9429-c21635d2617b.png">